code/_onclick/click.dm

```diff
@@ -304,10 +304,23 @@
 
 /**
  * Right click
+ *
+ * Used for Rightclick interactions, in similar fashion of AltClick.
+ * Returns RightClick().
```

Comment:
This is still not good documentation, it's just documenting an implementation detail.

What does the return value MEAN?